docs/platform/release-notes/tizen-5-5-m2.md

```diff
@@ -0,0 +1,641 @@
+  - Unicode variation selector support has been added.
+- Accessibility
+  - Universal switch AT-client has been tested.
+- rlottie
+  - Lots of vector properties have been added.
+    - Shape trim path (individual)
+    - Auto orient
+    - Spatial Bezier interpolation
+    - Reve across time
+    - Mask path
+    - Mask opacity
+    - Mask add
+    - Mask subtract
+    - Mask intersect
+    - Alpha matte
+    - Alpha inverted matte
+- Multi assistant FW
+  - Reference default wakeup engine has been added.
+  - New reference assistant such as google assistant has been added. 
+- Voice FW 
+  - New API set for voice control manager C# application have been added.
+  - New API set to be able to request to voice control engine to synthesize text to speech using its own voice have been added.
+  - New API set to set audio type on stt engine side have been added.
+  - New API set to be able to receive utterance status while user speaks using STT have been added.
+- Autofill Framework
+  - API to cancel the fill request has been added.
+- Sticker Framework
+  - A sticker framework that provides sticker information to application and retrieves sticker information in application has been provided.
+
+### Fixes
+
+- Memory consumption of rlottie has been improved.
+
+### Known Issues
+
+- Universal switch application AT-client has been tested only in mobile profile.
+
+
+## Multimedia framework
+
+### New and changed features
+
+- Sound Manager
+  - Audio ducking API set has been added.
+  - API set for preferred built-in device has been added.
+  - API to support network device has been added.
+- Audio IO
+  - Channel enum has been extended to support multi-channel recording.
+  - Format enum has been extended to support S32LE playback and recording.
+  - Deprecated symbols related to Unity have been added for compatibility.
+- Gstreamer-sharp
+  - C# binding code for gstreamer on Tizen has been regenerated.
+  - API to handle H/W resources and display has been added.
+  - Xamarin sample applications to provide how to make custom pipeline with Tizen gstreamer plugins have been added.
+  - Simple guideline for Tizen Xamarin application has been added.
+- Media Player
+  - API set for audio offloading has been added.
+  - API to query about supported format for push media stream has been added.
+  - Support for audio pcm extraction has been added.
+  - Support for setting audio codec type has been added.
+- Vision
+  - Deep Neural Network (DNN) based high level vision inference API set, such as image classification, object detection, face and its landmark detection, have been added.
+    - tensorflow-lite model data has been supported.
+    - caffe and tensorflow model data with OpenCV has been supported.
+- Media Controller
+  - API set to support subtitle, 360 mode, display mode, display rotation have been added.
+  - API set for ability has been added.
+  - New error for ability not supported has been added.
```

Comment:
ability is spec for some application.
there are abilities such as play stop, play pause, shuffle, repeat etc.
For example, music player application can support play stop/pause, shuffle etc.
but radio application can suppport those things.
it's ability.
Actually "New error" is "MEDIA_CONTROLLER_ERROR_ABILITY_LIMITED_BY_SERVER_APP".
each platform has it's own errors can return, and new error has been added recently.

ypedef enum {
	MEDIA_CONTROLLER_ERROR_NONE						= TIZEN_ERROR_NONE,						/**< Successful */
	MEDIA_CONTROLLER_ERROR_INVALID_PARAMETER		= TIZEN_ERROR_INVALID_PARAMETER,		/**< Invalid parameter */
	MEDIA_CONTROLLER_ERROR_OUT_OF_MEMORY			= TIZEN_ERROR_OUT_OF_MEMORY,			/**< Out of memory */
	MEDIA_CONTROLLER_ERROR_INVALID_OPERATION		= TIZEN_ERROR_INVALID_OPERATION,		/**< Invalid Operation */
	MEDIA_CONTROLLER_ERROR_FILE_NO_SPACE_ON_DEVICE		= TIZEN_ERROR_FILE_NO_SPACE_ON_DEVICE,	/**< No space left on device */
	MEDIA_CONTROLLER_ERROR_PERMISSION_DENIED		= TIZEN_ERROR_PERMISSION_DENIED,		/**< Permission denied */
	MEDIA_CONTROLLER_ERROR_ABILITY_LIMITED_BY_SERVER_APP	= MEDIA_CONTROLLER_ERROR_CLASS | 0x01,		/**< Limited by server application (since 5.5) */
} mc_error_e;

So, Can you guide me again?
"New error message has been added for ability not supported." is good to me.